libraries/AP_Scripting/applets/Bendy_ruler.lua

```diff
@@ -0,0 +1,871 @@
+local UPDATE_RATE_HZ = 10
```

Comment:
Aren't CAPITALs supposed to be reserved for GLOBALS?

Reply:
It’s a global considering it’s in the outermost scope of the script and a setting at the top. Marking it as a local is probably marginally better for performance.